setup.scripts/spamassassin/bayes_mysql.sql

```diff
@@ -24,7 +24,7 @@ CREATE TABLE IF NOT EXISTS `bayes_token` (
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
 CREATE TABLE IF NOT EXISTS `bayes_vars` (
-`id` int(11) NOT NULL,
+  `id` int(11) NOT NULL AUTO_INCREMENT,
```

Comment:
No longer works.

`ERROR 1075 (42000) at line 26: Incorrect table definition; there can be only one auto column and it must be defined as a key`